src/hooks/useKeyPress.js

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from 'react';
+
+const BACKSPACE = 8
+const SPACEBAR = 32
+
+const useKeyPress = callback => {
+
+  const [keyPressed, setKeyPressed] = useState();
+  useEffect(() => {
+    const downHandler = (e) => {
+      const {key, keyCode} = e;
+      if (keyCode === BACKSPACE) {
+        e.preventDefault();
+      }
+      if ((keyPressed !== key && key.length === 1) || keyCode === BACKSPACE || keyCode === SPACEBAR) {
```

Comment:
Could change `keyCode === BACKSPACE || keyCode === SPACEBAR` to `includes([BACKSPACE, SPACEBAR], keycode)`

Reply:
nice 👍